paddle/fluid/platform/ngraph_helper.h

```diff
@@ -189,6 +189,22 @@ inline void TrimTrailingSingularDims(ngraph::Shape* shape) {
     }
   }
 }
+
+ngraph::element::Type GetNgType(paddle::framework::proto::VarType::Type dtype) {
```

Comment:
some cloud use this function I think. this function is common.
 
like this:
https://github.com/PaddlePaddle/Paddle/blob/dfdcd91869595c7f068767e15f05f325d8a1de9c/paddle/fluid/operators/ngraph/ops/fill_constant_op.h#L40-L54

Reply:
Thank you for your suggestion.